server/constants/constants.go

```diff
@@ -155,6 +155,7 @@ const (
 	ErrorGetStates                        = "Error in getting the states"
 	ErrorUpdateState                      = "Error in updating the state"
 	ErrorACLRestrictsRecordRetrieval      = "ACL restricts the record retrieval"
+	ErrorCommandInvalidNumberOfParams     = "One or more fields are missing to run the command. Please run `/servicenow help` for more information."
```

Comment:
i think we can change this message to something like this give your thoughts:-
```suggestion
	ErrorCommandInvalidNumberOfParams     = "Some field(s) are missing to run the command. Please run `/servicenow help` for more information."
```
we could event get rid of some in the starting